config/development.yaml

```diff
@@ -82,19 +82,27 @@ dcRedirectionPolicy:
   toDC: ""
 
 archival:
-  status: "enabled"
-  enableReadFromArchival: true
-  defaultBucket: "cadence-development"
-  filestore:
-    storeDirectory: "/tmp/development/blobstore/"
-    defaultBucket: "cadence-development"
-    customBuckets:
-      - "custom-bucket-1"
-      - "custom-bucket-2"
-#  s3store:
-#    region: "us-east-1"
-#    endpoint: "http://127.0.0.1:4572"
-#    s3ForcePathStyle: true
+  history:
+    status: "disabled"
```

Comment:
I need to disable it for now since the reset part of the system is still using blobstore, which will be nil.